rosbag2_py/src/rosbag2_py/_transport.cpp

```diff
@@ -175,6 +177,30 @@ class Recorder
   }
 };
 
+// Simple wrapper to read the output config YAML into structs
+void bag_rewrite(
```

Comment:
TODO: I really should add at least a simple smoke test for this function, to make sure that it can load a reasonable sample YAML file

Reply:
Yeah, decided to go back to draft while adding this test. Any comments on the feature code welcome though!